spec.html

```diff
@@ -100,9 +100,8 @@ <h1>HostResizeArrayBuffer ( _buffer_, _newByteLength_ )</h1>
       <p>The implementation of HostResizeArrayBuffer must conform to the following requirements:</p>
       <ul>
         <li>The abstract operation does not detach _buffer_.</li>
-        <li>The abstract operation may complete normally or abruptly.</li>
         <li>If the abstract operation completes normally with ~handled~, _buffer_.[[ArrayBufferByteLength]] is _newByteLength_.</li>
-        <li>The return value is either ~handled~ or ~unhandled~.</li>
+        <li>The return value is either ~handled~, ~unhandled~, or an abrupt throw completion.</li>
```

Comment:
I'd reword this as "The abstract operation may return a normal completion with a value of \~handled\~, a normal completion with a value of \~unhandled\~, or an abrupt throw completion.", as well as move it back up to the first requirement.

Reply:
I can see how returning normal completions is good for consistency's sake (even though ReturnIfAbrupt works across mixed types). I'm not so sure about the use of "may," though. How do you feel about: "The abstract operation must return either a normal completion with a value of \~handled~, a normal completion with a value of \~unhandled~, or an abrupt throw completion." ?

Reply:
That use of "must" sgtm.

Reply:
Great! Also, I found [the NormalCompletion abstract operation](https://tc39.es/ecma262/#sec-normalcompletion) and decided to use it here for brevity and specificity (since "a normal completion with a value of \~handled~" is ambiguous in regards to the completion record's [[Target]] slot).